### PR TITLE
structure examples directory with multiple subdirectories

### PR DESCRIPTION
**Pull request by allaffa**
… example

### CODE REVIEW

File: tests/test_config.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 @pytest.mark.mpi_skip()
 def pytest_config(config_file):
 
-    config_file = os.path.join("examples", config_file)
+    config_file = os.path.join("examples/dft_lsms", config_file)
```
Comment (streeve) [inline tests/test_config.py:20]: Maybe just `lsms` for now? If we expand to examples outside DFT we can add another level to the hierarchy

Comment (allaffa) [reply tests/test_config.py]: Done.

**Comment by pzhanggit** [comment]
Need to update example.py file.

**Review by pzhanggit — APPROVED** [review]

**Comment by pzhanggit** [comment]
> In the QM9 PR I've named `examples/qm9/qm9.json`. I would prefer `lsms/lsms.json` - I assume you disagree :)

Have to reply, lolllll

**Comment by allaffa** [comment]
> In the QM9 PR I've named `examples/qm9/qm9.json`. I would prefer `lsms/lsms.json` - I assume you disagree :)

Done. I do not know the difference between agreeing and disagreeing anymore. :)

**Review by streeve — APPROVED** [review]

**Review by streeve — CHANGES_REQUESTED** [review]
Approved, but can you squash these commits?

**Pull request merged**